src/components/resumption/include/resumption/last_state_wrapper_impl.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, Ford Motor Company
+ * Copyright (c) 2018, Ford Motor Company
```

Comment:
@YarikMamykin 2019

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/61f8fecf23bd114017777ce8d90cdb0fb952edf1